src/components/flat-table/sort/sort.component.tsx

```diff
@@ -26,10 +30,9 @@ const Sort = ({ children, onClick, sortType }: SortProps) => {
 
   return (
     <>
-      <span hidden id={id.current}>
-        {children}
-        {sortType ? `, sort type ${sortType}` : ", sortable"}
-      </span>
+      <Typography screenReaderOnly id={id.current}>
+        {locale.sort.accessibleName(children, sortType)}
```

Comment:
I thought the intention was to change `children` to be `string`? Currently we're potentially passing a node here and are having to include it in the translation type

https://github.com/Sage/carbon/pull/6592#discussion_r1497743459

Reply:
The type has been changed from a `node` -> `string` in the second commit. Commit includes a breaking change too, to notify consumers